packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -107,9 +107,6 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
       classNames.bodyContent,
       {
         padding: 20
-      },
-      isWide && {
-        maxWidth: 302
```

Comment:
worried that this is breaking the "isWide" prop. Are we doing something to make this work properly?

Reply:
Hmm what do you mean it's breaking the "isWide" prop? the examples all work the same still if that's what you mean - previous comments I have screenshots showing this

Reply:
ah, i see up on line 42 that isWide changes the content width

```
isWide && {
      maxWidth: 456
    }
```

Ok, i was just wondering why this maxWidth was dissapearing.